scripts/dockerfiles/Dockerfile.main-debug-fs

```diff
@@ -0,0 +1,4 @@
+FROM amazon/aws-for-fluent-bit:main-debug-base
+COPY ./scripts/dockerfiles/Dockerfile.main-debug-base /Dockerfile.2.main-debug-base
+
+CMD /fluent-bit/bin/fluent-bit -c /fluent-bit/etc/fluent-bit.conf; echo "Waiting 2 minutes for EFS core file transfers"; sleep 120
```

Comment:
Oh so the core files weren't working on EFS because they are big and take a while???

Reply:
That's right! In my testing the core files wouldn't actually get synced to EFS unless a delay was added.